tests/service/data/SyntaxTree/Type/Abstract Class With Invalid Constructs 01.fs

```diff
@@ -0,0 +1,13 @@
+// Expected: Multiple warnings for module and type
+module Module
+
+[<AbstractClass>]
+type AbstractBase() =
+    abstract member Method : int -> int
+    
+    module InvalidModule = 
+        let helper = 10
+    
+    type InvalidType = string
+    
+    default _.Method(x) = x * 2
```

Comment:
@edgarfgp Could you please try splitting the tests into smaller ones, so it would be easier to reason about the baselines? Let's try to make more simpler tests instead, so intent of each one is clear.

There're also details that don't seem to have any useful effect but add noise to the baselines:
Does the `AbstractClass` attribute add something specific to this case?
Is the primary constructor needed here to verify the parsing of the members?

Please try to keep the parsing tests minimal 🙂

Reply:
Done.